.github/workflows/be-dev-deploy-closed-prs.yml

```
@@ -1,11 +1,20 @@
 name: BE Dev Deploy Closed Prs
 
+#on:
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)